cypress-shared/support/adyen/graphql.js

```diff
@@ -0,0 +1,98 @@
+import { FAIL_ON_STATUS_CODE } from '../common/constants'
+import { updateRetry } from '../common/support'
+
+const config = Cypress.env()
+
+// Constants
+const { vtex } = config.base
+
+export function updateAdyenConnectorSettings({
+  merchantAccount = vtex.merchantAccount,
```

Comment:
why are we having them in function parameters
If we are not updating them then can u please put this inside function body